elephas/parameter/server.py

```diff
@@ -134,9 +134,10 @@ def base_constraint(a): return a
                 self.lock.release()
             return 'Update done'
 
-        self.app.run(host='0.0.0.0', debug=self.debug, port=self.port,
-                     threaded=self.threaded, use_reloader=self.use_reloader)
-
+        # self.app.run(host='0.0.0.0', debug=self.debug, port=self.port,
```

Comment:
wait, if we replace `host='0.0.0.0'` above with `host=self.master_url.split(':')[0]` that should do pretty much the same thing. werkzeug is used under the hood after all afaik.